spec/controllers/complete_registration_controller_spec.rb

```diff
@@ -43,7 +43,8 @@
       end
 
       it "@user should be assigned" do
-        post :update, :id => "known"
+        post :update, { id: "known",
```

Comment:
Per our Slack conversation, let's bump the line limit to 85 characters. The we can unwrap lines like this.

Reply:
Had to bump to 90.  85 was not enough.